__tests__/hooks/useFeedHeader.test.js

```diff
@@ -54,7 +54,7 @@ describe('Feed Header should run correctly', () => {
     act(() => {
       result.current.onBackNormalUser();
     });
-    expect(mockedNavigate).toHaveBeenCalled();
+    expect(mockedGoBack).toHaveBeenCalled();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `mockedNavigate` to `mockedGoBack` seems appropriate if the function's behavior has been modified to go back instead of navigating to a new screen. However, it would be beneficial to add an assertion to check the number of times `mockedGoBack` is called to ensure that it is not being invoked more than expected.
```diff
-    expect(mockedGoBack).toHaveBeenCalled();
+    expect(mockedGoBack).toHaveBeenCalledTimes(1);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->